src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsCardView/HomeStatisticsCardViewModel.swift

```diff
@@ -77,8 +77,26 @@ class HomeStatisticsCardViewModel {
 		} else {
 			primaryTitle = String(format: AppStrings.Statistics.Card.Incidence.date, "")
 		}
-
 		primarySubtitle = AppStrings.Statistics.AddCard.localCardPrimarySubtitle
```

Comment:
I think this one is outdated and should just say "Neuinfektionen" now

Reply:
Fixed the text